udata/tests/test_routing.py

```diff
@@ -164,6 +164,18 @@ def test_redirect_to_normalized_utf8_slug(self, client):
         assert_redirects(client.get('/model/slüg'), url)
 
 
+class SlugAndIdTest(AsSlugMixin):
+    model = SlugTester
+    converter = SlugTesterConverter
+
+    def test_url_hijack(self, client):
```

Comment:
Can you test the UUID-in-slug thing also?

Reply:
Done